src/Analysis/Ast/Impl/Analyzer/Definitions/IExpressionEvaluator.cs

```diff
@@ -90,4 +91,4 @@ public interface IExpressionEvaluator {
         /// </summary>
         IPythonType UnknownType { get; }
     }
-}
+}
```

Comment:
Here's an editor formatting difference; make sure you're setup to have files end with newlines.

Reply:
I'll get to it once I am closer to finishing